src/app/Console/Commands/Install.php

```diff
@@ -96,13 +96,10 @@ private function createUsers()
         $userClass = config('backpack.base.user_model_fqn', 'App\Models\User');
         $userModel = new $userClass();
 
-        // Count current users
-        $currentUsers = $userModel->count();
-
         $this->newLine();
         $this->infoBlock('Creating an admin:', 'Step 2');
         $this->note('Quickly jump in your admin panel, using the email & password you choose here.');
-        $this->note('Currently there '.trans_choice("{0} are <fg=blue>no users</>|{1} is <fg=blue>1 user</>|[2,*] are <fg=blue>$currentUsers users</>", $currentUsers).' in the database.');
```

Comment:
@promatik - heads-up, I like the old message better, to be honest. And let's keep the `count()` please, I liked it. Maybe the end result looks something like this?

> Currently there are _2 users_ in the database.

The expanded version would be:

> Currently there are _2 users_ in the database (`users` table, as defined in `App\Models\User`).

But I think that's too much information for this point in time. Let's keep it simple here.

Reply:
Reverted 👌